docs/contributing.rst

```diff
@@ -2,12 +2,705 @@
 
 .. _contributing:
 
-
+*******************
 Contributor's Guide
-===================
+*******************
+
+Welcome to the Contributor's Guide for UXarray!
+
+Welcome to the team! If you are reading this document, we hope that
+you are already or soon-to-be a UXarray contributor, please keep reading!
+
+1. Overview
+===========
+
+UXarray is a community-owned, open-development effort that is a result
+of the collaboration between `Project Raijin <https://raijin.ucar.edu/>`_
+and DOE’s `SEATS Project <https://seatstandards.org/>`_. Even though the
+UXarray team has initiated and been expanding this package, outside
+contributions are welcome and vital for its long-term sustainability.
+Therefore, we invite other community members to become part of this
+collaboration at any level of contribution.
+
+1.1. Why the name "UXarray"?
```

Comment:
This seems like it belongs in the README instead of the Contributors guide.  (Potential contributors should read the README)

Reply:
We can assume they've already read the README if they're reading the Contributor's Guide.

Reply:
Or just add that to the instructions.

Reply:
This makes sense. Moved this section to README, which also led me to adjust some headings as well as fix the use of the words "Uxarray"  in README.